lld/COFF/DriverUtils.cpp

```diff
@@ -246,6 +246,22 @@ void LinkerDriver::parseAligncomm(StringRef s) {
       std::max(ctx.config.alignComm[std::string(name)], 1 << v);
 }
 
+void LinkerDriver::parseDosStub(StringRef path) {
+  std::unique_ptr<MemoryBuffer> stub =
+      CHECK(MemoryBuffer::getFile(path), "could not open " + path);
+  size_t bufferSize = stub->getBufferSize();
+  const char *bufferStart = stub->getBufferStart();
+  // MS link.exe compatibility:
+  // 1. stub must greater than or equal to 64 bytes
```

Comment:
I didn't mean for you to remove "be" on this line though :)

Reply:
it's fixed, thank you for pointing out!